sources/packages/api/src-sql/InstitutionUsers/Add-col-institution-user-guid.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE "institution_users" ADD COLUMN IF NOT EXISTS "guid" VARCHAR(100) NULL;
```

Comment:
Please consider changing the column name to something that indicates that it is the user_guid.